css/github-embed.css

```diff
@@ -5,7 +5,8 @@
 	width: 80%;
 	margin: auto;
 	margin-bottom: 1em;
-	background-image: url('https://github.com/github/media/raw/master/octocats/blacktocat-32.png');
+	background-image: url('data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjEwMTQiIHdpZHRoPSIxMDI0IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPjxwYXRoIGQ9Ik01MTIgMEMyMjkuMjUyIDAgMCAyMjkuMjUxOTk5OTk5OTk5OTUgMCA1MTJjMCAyMjYuMjUxIDE0Ni42ODggNDE4LjEyNiAzNTAuMTU1IDQ4NS44MTMgMjUuNTkzIDQuNjg2IDM0LjkzNy0xMS4xMjUgMzQuOTM3LTI0LjYyNiAwLTEyLjE4OC0wLjQ2OS01Mi41NjItMC43MTgtOTUuMzE0LTEyOC43MDggMjMuNDYtMTYxLjcwNy0zMS41NDEtMTcyLjQ2OS02MC4zNzMtNS41MjUtMTQuODA5LTMwLjQwNy02MC4yNDktNTIuMzk4LTcyLjI2My0xNy45ODgtOS44MjgtNDMuMjYtMzMuMjM3LTAuOTE3LTMzLjczNSA0MC40MzQtMC40NzYgNjkuMzQ4IDM3LjMwOCA3OC40NzEgNTIuNzUgNDUuOTM4IDc3Ljc0OSAxMTkuODc2IDU1LjYyNyAxNDguOTk5IDQyLjUgNC42NTQtMzIuOTk5IDE3LjkwMi01NS42MjcgMzIuNTAxLTY4LjM3My0xMTMuNjU3LTEyLjkzOS0yMzMuMjItNTYuODc1LTIzMy4yMi0yNTMuMDYzIDAtNTUuOTQgMTkuOTY4LTEwMS41NjEgNTIuNjU4LTEzNy40MDQtNS4yMi0xMi45OTktMjIuODQ0LTY1LjA5NSA1LjA2My0xMzUuNTYzIDAgMCA0Mi45MzctMTMuNzQ5IDE0MC44MTEgNTIuNTAxIDQwLjgxMS0xMS40MDYgODQuNTk0LTE3LjAzMSAxMjguMTI0LTE3LjIyIDQzLjQ5OSAwLjE4OCA4Ny4zMTQgNS44NzQgMTI4LjE4OCAxNy4yOCA5Ny42ODktNjYuMzExIDE0MC42ODYtNTIuNTAxIDE0MC42ODYtNTIuNTAxIDI4IDcwLjUzMiAxMC4zNzUgMTIyLjU2NCA1LjEyNCAxMzUuNDk5IDMyLjgxMSAzNS44NDQgNTIuNjI2IDgxLjQ2OCA1Mi42MjYgMTM3LjQwNCAwIDE5Ni42ODYtMTE5Ljc1MSAyNDAtMjMzLjgxMyAyNTIuNjg2IDE4LjQzOSAxNS44NzYgMzQuNzQ4IDQ3LjAwMSAzNC43NDggOTQuNzQ4IDAgNjguNDM3LTAuNjg2IDEyMy42MjctMC42ODYgMTQwLjUwMSAwIDEzLjYyNSA5LjMxMiAyOS41NjEgMzUuMjUgMjQuNTYyQzg3Ny40MzYgOTI5Ljk5OCAxMDI0IDczOC4xMjYgMTAyNCA1MTIgMTAyNCAyMjkuMjUxOTk5OTk5OTk5OTUgNzk0Ljc0OCAwIDUxMiAweiIgLz48L3N2Zz4=');
```

Comment:
SVG support isn't universal, so while I'm open to swapping in an SVG for browsers that support it - I'd still like a fallback for those that don't (Realistically I'm talking IE8 here). Can you update the PR with an appropriate fallback?